view/theme/frio/templates/head.tpl

```diff
@@ -39,11 +39,15 @@
 <link rel="shortcut icon" href="{{$shortcut_icon}}" />
 <link rel="apple-touch-icon" href="{{$touch_icon}}"/>
 
-<meta name="apple-mobile-web-app-capable" content="yes" /> 
+<meta name="apple-mobile-web-app-capable" content="yes" />
+<script>
```

Comment:
I don't know how the other prefer it for the regular templates but for frio it would be nice if you could move your js code into frios theme.js (or create a file frio/js/mobile.js and include it in frios head.tpl). The future plan for frio is to avoid putting js directly in the templates as far as possible because it would be easier to debug.